src/api/v1/db/oracledb/setupDb.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE PIZZA_INGREDIENTS;
+DROP TABLE PIZZAS;
+DROP TABLE INGREDIENTS;
+DROP TABLE DOUGHS;
+
+CREATE TABLE "DOUGHS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL ENABLE, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"GRAMS_FLOUR" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_WATER" NUMBER(*,0) NOT NULL ENABLE, 
+	"FLOUR_TYPE" VARCHAR2(256 BYTE) NOT NULL ENABLE, 
+	"WATER_TEMP" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_YEAST" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_SALT" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_SUGAR" NUMBER(*,0) NOT NULL ENABLE, 
+	"GRAMS_OLIVE_OIL" NUMBER(*,0) NOT NULL ENABLE, 
+	"BULK_FERMENT_TIME" NUMBER(*,0) NOT NULL ENABLE, 
+	"PROOF_TIME" NUMBER(*,0) NOT NULL ENABLE, 
+	"SPECIAL_INSTRUCTIONS" VARCHAR2(512 BYTE), 
+	 CONSTRAINT "DOUGHS_PK" PRIMARY KEY ("ID") ENABLE
```

Comment:
It seems like there's a bunch of ways to define constraints in oracle sql. Not sure if there's a preferred method.